app/controllers/forest_liana/resources_controller.rb

```diff
@@ -22,7 +22,7 @@ def show
       getter.perform
 
       render serializer: nil, json:
-        serialize_model(getter.record, include: includes)
+        serialize_model(getter.record, include: includes(getter))
```

Comment:
Breaks the `resource#show`, because the `getter` does not have an `includes` method:
<img width="834" alt="screen shot 2016-11-01 at 23 25 19" src="https://cloud.githubusercontent.com/assets/664261/19918729/f8847414-a08a-11e6-8b5e-4f93ab32fcc0.png">